packages/shared/src/components/MainFeedLayout.tsx

```diff
@@ -254,20 +256,38 @@ export default function MainFeedLayout({
     return isLaptop ? cloudinary.feed.bg.laptop : cloudinary.feed.bg.tablet;
   };
 
-  const FeedPageComponent = shouldUseFeedLayoutV1 ? FeedPageLayoutV1 : FeedPage;
+  const getPadding = () => {
+    if (isV1Search) {
+      return '!pt-2';
+    }
+
+    return shouldUseFeedLayoutV1 && '!pt-0';
+  };
 
   return (
-    <FeedPageComponent className="relative">
-      {searchVersion === SearchExperiment.V1 && !isFinder && (
+    <FeedPage className={classNames('relative', getPadding())}>
+      {isV1Search && !isFinder && (
         <img
-          className="absolute left-0 top-0 w-full max-w-[58.75rem]"
+          className={classNames(
+            'absolute left-0 top-0 w-full laptop:max-w-[58.75rem]',
+            shouldUseFeedLayoutV1 && 'laptop:left-1/2 laptop:-translate-x-1/2',
+          )}
           src={getImage()}
           alt="Gradient background"
         />
       )}
       {isSearchOn && search}
-      {feedProps && <Feed {...feedProps} />}
+      {feedProps && (
+        <Feed
+          {...feedProps}
+          className={
+            (shouldUseFeedLayoutV1 || isV1Search) &&
+            !isFinder &&
+            'laptop:!max-w-[36.5rem]'
```

Comment:
I'm not sure I understand why this is needed

Reply:
If you keep this, the content will be maxed out, which should not be the case.

Reply:
This is from Figma's artboard.

![Screenshot 2024-01-04 at 6 58 44 PM](https://github.com/dailydotdev/apps/assets/13744167/3a7e0887-370b-43ac-90ec-94afc1248088)

With regards to the padding, we have the right spacing as seen below. It is just that the margin was used over the padding but should not make a difference when either is used.

![Screenshot 2024-01-04 at 7 00 29 PM](https://github.com/dailydotdev/apps/assets/13744167/d2bc45cc-d0f4-4bd1-baf5-7565462f067b)

Reply:
We even have a bigger padding than what is in Figma.

![Screenshot 2024-01-04 at 7 02 20 PM](https://github.com/dailydotdev/apps/assets/13744167/0c00d7ee-3231-4b04-b9b4-c86e73138012)

Reply:
Yup, that bigger margin is because of this `laptop:!max-w-[36.5rem]`. That's why I'm suggesting to remove it, and just add paddings/margins whatever works best.

Reply:
Conversation continued here https://github.com/dailydotdev/apps/pull/2464#discussion_r1441600408

Reply:
It is basically the same thing, by applying paddings which will equate to the total margin displayed here.